core/services/workflows/models_test.go

```diff
@@ -186,6 +186,31 @@ targets:
 				"a-target": {},
 			},
 		},
+		{
+			name: "non-trigger step with no refs",
+			yaml: `
+triggers:
+  - type: "a-trigger"
+  - type: "a-second-trigger"
+actions:
+  - type: "an-action"
+    ref: "an-action"
+    inputs:
+      hello: "world"
```

Comment:
@bolekk This step; ref = reference to another step

Reply:
(BTW, I realise this is super confusing; have tightened up the test and the error that we throw to be clearer)

Reply:
thanks :)